src/main/java/net/coderbot/iris/mixin/rendertype/MixinItemBlockRenderTypes.java

```diff
@@ -0,0 +1,60 @@
+package net.coderbot.iris.mixin.rendertype;
+
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.block.state.BlockState;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.coderbot.iris.layer.GbufferProgram;
+import net.coderbot.iris.layer.IrisRenderTypeWrapper;
+import net.coderbot.iris.layer.UseProgramRenderStateShard;
+import net.coderbot.iris.pipeline.HandRenderer;
+import net.minecraft.client.renderer.ItemBlockRenderTypes;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.client.renderer.Sheets;
+import net.minecraft.client.renderer.texture.TextureAtlas;
+
+@Mixin(ItemBlockRenderTypes.class)
```

Comment:
This entire class could be removed if my comments in MixinRenderType are accurate.

Reply:
Doesn't seem so, sadly.

Reply:
It should be. You might need to return HAND_TRANSLUCENT in GbufferPrograms#refine if the program is TRANSLUCENT_TERRAIN.